### PR TITLE
Update compat with AIO Preview 0.8.0

### PR DESCRIPTION
**Pull request by katriendg**
## Purpose

* Did some generic security updates (dependabot, pinned versions in GH Actions)
* Update the Flux folder to include deployment of OpenTelemetry collector (instead of manual YAML)
* Update onboarding of cluster to AIO with version 0.8.0 Preview

## Does this introduce a breaking change?

```
[x] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[x] Other... Please describe: updates to several flows to support new preview version
```

## How to Test
*  See Readme

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
 	// Features to add to the dev container. More info: https://containers.dev/features.
 	"features": {
 		"ghcr.io/devcontainers/features/azure-cli:1": {
-			"extensions": "connectedk8s,k8s-extension,azure-iot-ops,k8s-configuration"
+			"version": "2.64.0",
```
Comment (bindsi) [inline .devcontainer/devcontainer.json:13]: Thanks for using my workaround suggestion

File: .github/dependabot.yaml
Comment (bindsi) [inline .github/dependabot.yaml:1]: I should include that in my repos as well!!!

File: README.md [hunk 2]
```diff
@@ -4,71 +4,74 @@ This sample is currently work in progress.
...
 
-If you’re looking for a default observability setup and wish to analyze observability data using Azure Monitor, follow [this documentation](https://learn.microsoft.com/en-us/azure/iot-operations/monitor/howto-configure-observability).
+If you’re looking for a default observability setup and wish to analyze observability data using Azure Arc extensions and Azure Monitor in the cloud, follow [this documentation](https://learn.microsoft.com/en-us/azure/iot-operations/monitor/howto-configure-observability).
 
 > [!WARNING]
-Azure IoT Operations is still in Preview. Any parts of sample could stop working as the product evolves towards General Availability. The current version is pinned in the dev container, and is Azure IoT Operations Preview v0.6.0.
+> Azure IoT Operations is still in Preview. Any parts of sample could stop working as the product evolves towards General Availability. The current version is pinned in this dev container is Azure IoT Operations Preview v0.8.0.
```
Comment (bindsi) [inline README.md]: maybe use the real version number v0.8.32...just my 2 cents

File: README.md [hunk 3]
```diff
@@ -4,71 +4,74 @@ This sample is currently work in progress.
...
 
-```
+   ```
 
-* The newly created file `./temp/envvars.sh` should now be open in the Code editor.
-* Update the variables in the file according to these details:
-  * For the resource names, use your preferred Azure resource names in the variable contents. An example using the recommended resource abbreviations provided in the comments
-  * `ARC_CUSTOMLOCATION_OID`: retrieve the unique Custom location Object ID for your tenant by running `az ad sp show --id bc313c14-388c-4e7d-a58e-70017303ee3b --query id -o tsv`
-  * `LOCATION`: make sure you choose something from the supported list of regions as documented in [Deploy Azure IoT Operations, see Location table](https://learn.microsoft.com/en-us/azure/iot-operations/get-started/quickstart-deploy?tabs=codespaces#connect-a-kubernetes-cluster-to-azure-arc)
+1. The newly created file `./temp/envvars.sh` should now be open in the Code editor.
```
Comment (bindsi) [inline README.md]: ...now be opened...

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+* The location of the affected source code (tag/branch/commit or direct URL)
+* Any special configuration required to reproduce the issue
+* Step-by-step instructions to reproduce the issue
+* Proof-of-concept or exploit code (if possible)
+* Impact of the issue, including how an attacker might exploit the issue
+
+This information will help us triage your report more quickly.
+
+If you are reporting for a bug bounty, more complete reports can contribute to a higher bounty award. Please visit our [Microsoft Bug Bounty Program](https://aka.ms/security.md/msrc/bounty) page for more details about our active programs.
+
```
Comment (bindsi) [inline SECURITY.md:32]: Really liked that section...need to get this template too

**Review by bindsi — APPROVED** [review]
Looks really good to me!!! Good job👌

File: SECURITY.md (hunk 4 above)
Comment (katriendg) [reply SECURITY.md:32]: Part of a standard template :)

File: README.md (hunk 2 above)
Comment (katriendg) [reply README.md]: Correct, updated

**Comment by katriendg** [comment]
@bindsi Thanks for your review. I realized some updates were missing in the observability description page so now updated that too. Could you take a look at the page https://github.com/Azure-Samples/azure-edge-extensions-aio-observability-gitops/pull/19/files#diff-1bf3e4d49c56a8a6c46aa5facfa5cf362f450f649e5b89e04b48fbaf4892dd24

File: docs/observability-setup.md [hunk 5]
```diff
@@ -29,24 +29,27 @@ The local edge observability stack is composed of the following components:
...
 
 In the default configuration, the collected metrics are exposed by the collector on port 8889. To view them, run the command `kubectl port-forward svc/aio-otel-collector 8889:8889 -n azure-iot-operations` and then access http://localhost:8889/metrics. Additionally, logs and traces are logged to the console output of the collector.
 
-The OpenTelemetry Collector is highly configurable, allowing for the definition of multiple receivers, processors, and exporters. Based on [this documentation](https://learn.microsoft.com/en-us/azure/iot-operations/deploy-custom/howto-helm#deploy-a-helm-chart-with-override-values), [a Bicep script](../deploy/templates/aio-otel-collector-update.bicep) was created to override the base values with the following [overlay values](../deploy/templates/aio-otel-collector-overlay-values.yml).
+The OpenTelemetry Collector is highly configurable, allowing for the definition of multiple receivers, processors, and exporters.
 
-The overlay values include the addition of the following components:
+The configuration we use for this sample include:
```
Comment (bindsi) [inline docs/observability-setup.md]: he/she/it --> includes

**Review by bindsi — APPROVED** [review]
Looks good :-)

**Pull request merged**